test/src/tests-batchUploader_2.ts

```diff
@@ -3,8 +3,8 @@ import Utils from './config/utils';
 import { expect } from 'chai';
 import _BatchValidator from '../../src/mockBatchCreator';
 import fetchMock from 'fetch-mock/esm/client';
-import { IMParticleInstanceManager } from '../../src/mparticle-instance-manager';
 import { CustomEvent } from '@mparticle/event-models';
+import { IMParticleInstanceManager } from '../../src/sdkRuntimeModels';
```

Comment:
nit: very minor, but move back to line 6 to reduce line changes?  same across a few of the below files